fedimint-client/src/transaction/sm.rs

```diff
@@ -262,6 +262,10 @@ mod tests {
             &self.fake_peers
         }
 
+        fn with_module(&self, _id: ModuleInstanceId) -> DynFederationApi {
+            todo!()
```

Comment:
AFAIR, `todo!()` is temporary "I'm working on this PR". For a long term stuff `unimplemented!()` is better.